### PR TITLE
LBSD-657 Implement about page modal and menu

### PR DESCRIPTION
**Pull request by vladnicoara**

### CODE REVIEW

File: client/app/styles/less/liveblog.less [hunk 1]
```diff
@@ -67,4 +67,65 @@
...
+.about-modal {
+    top: 115px!important;
+    .modal-dialog {
+        width: 662px!important;
+    }
+    p {
+        margin-top: 20px;
+    }
+}
+.about-page-text {
```
Comment (vied12) [inline client/app/styles/less/liveblog.less]: Is there any reason why here you are outside of the `.about-modal` scope ?

**Comment by vied12** [comment]
LGTM otherwise

**Comment by nistormihai** [comment]
LGTM

File: client/app/index.html [hunk 2]
```diff
@@ -23,8 +23,61 @@
...
+                    <img src="images/lb-logo-about.png" alt="Superdesk Logo" class="pull-right">
+                </div>
+            </div>
+            <div class="about-page-text">
+                <p>
+                    <div style="font-weight: 700" translate>
+                        Live Blog 3.0 beta-1
+                    </div>
+                    <div translate>
+                        released on 2015/09/22
```
Comment (hlmnrmr) [inline client/app/index.html]: Let's use camel case here...

File: client/app/index.html [hunk 3]
```diff
@@ -23,8 +23,61 @@
...
+                    </div>
+                </p>
+                <p translate>
+                    Sourcefabric's <a href="https://www.sourcefabric.org/en/liveblog/" target="_blank">Live Blog</a> is an open source web app that enables journalists to provide immediate and ongoing coverage on rapidly evolving news events. Live Blog is powered by the <a href="https://www.sourcefabric.org/en/superdesk/" target="_blank">Superdesk platform</a> and developed with love.
```
Comment (hlmnrmr) [inline client/app/index.html]: The "Superdesk platform" link should point to http://www.superdesk.org/ instead...

**Comment by hlmnrmr** [comment]
:+1:

**Pull request merged**